header.php

```diff
@@ -47,6 +47,7 @@ function h2($title)
         <link rel='stylesheet' type='text/css' href="//fonts.googleapis.com/css?family=Ubuntu:400,700,400italic"/>
         <link rel="stylesheet" type="text/css" href="<?php echo style('style.css') ?>?2017-09-15-0000" title="default" media="screen,print"/>
         <link rel="stylesheet" type="text/css" href="<?php echo script('highlight/styles/github.css') ?>"/>
+        <link rel="canonical" href="<?php echo $redirect ?>"/>
```

Comment:
Maybe have a check around this (similar to the doxygen one), as this header data is also used for every other site and not all of them are versioned.